scripts/pasha_main.coffee

```diff
@@ -127,17 +129,19 @@ module.exports = (robot) ->
             "#{botName} status: display or set prio1 status, " +
             "see '#{botName} status help' for details"
         robot.receive(new TextMessage(msg.message.user,
-            "#{botName} changelog helpFromMain"))
+            "#{botName} changelog help_from_main"))
         robot.receive(new TextMessage(msg.message.user,
```

Comment:
these kind of errors are already fixed in master and they don't belong to this feature anyways